third_party/openmetadata/install_OM.sh

```diff
@@ -1,39 +1,82 @@
 #!/bin/bash
 
+OPERATION=install
 OM_VERSION=0.12.1
+K8S_TYPE=kind
 export OPENMETADATA_AIRFLOW_VERSION=0.12.1
 export OPENMETADATA_HELM_CHART_VERSION=0.0.39
 
 export FYBRIK_BRANCH="${FYBRIK_BRANCH:-master}"
 export FYBRIK_GITHUB_ORGANIZATION="${FYBRIK_GITHUB_ORGANIZATION:-fybrik}"
 
-if [ $# -gt 1 ]; then
-    echo "Usage: "$0" [open-metadata-version]"
-    exit -1
+if [ $# -gt 3 ]; then
+    echo "Usage: . ./install_OM.sh [install/getFiles/cleanup] [ibm-openshift/kind] [open-metadata-version]"
```

Comment:
I'd print the default values of ibm-openshift/kind and OMD version, plus we can add the help operation